demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1291,7 +1291,7 @@ def run_validation_using_git(self):
                 ) as graph_validator:
                     validation_results.add(graph_validator.is_valid_content_graph())
 
-        validation_results.add(self.validate_modified_files(modified_files))
+        validation_results.add(self.validate_modified_files(modified_files.union(old_format_files)))
```

Comment:
if `old_format_files` are unified, care to rename it to `unified_integrations`? For readability.

Reply:
Rename what?
The `validate_modified_files` function name?

Reply:
`modified_files` and `old_format_files` remain the same separate sets, they aren't unified.
I just send `validate_modified_files` a newly generated unified set of both because I want it to run on both of these sets.

(using `union` creates a new set by combining two, it doesn't alter the original sets)